Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1

```diff
@@ -384,7 +471,7 @@ function Set-ClusterProtectionLeveltoEncryptandSignforServiceFabric
             if($userInput -ne "Y")
             {
                 Write-Host "Cluster Protection level will not be changed to EncryptAndSign on Service Fabric(s) in the Subscription. Exiting..." -ForegroundColor $([Constants]::MessageType.Warning)
-                break
```

Comment:
Not able to comment above to commenting it here:

line 427: $ServiceFabricWithoutEncryptandSign | Format-Table -Property $colsProperty -Wrap


put it in if(-not $Autoremediation) condition and also make it more understandable
Reference:

if(-not $AutoRemediation)
    {
        Write-Host "Container Registry(s) without security scanning enabled is as follows:"
        $containerRegistryWithoutSecurityScanningEnabled | Format-Table -Property $colsProperty -Wrap
        Write-Host $([Constants]::SingleDashLine)
    }

Reply:
Write-Host $([Constants]::DoubleDashLine)
    Write-Host "[Step 3 of 4] Backing up Service Fabric(s) details..."
    Write-Host $([Constants]::SingleDashLine)

=> remove double dash lines, double dash lines are only to be used whenever script execution might possibly end.
=> Write-Host "[Step 3 of 4] Back up Service Fabric(s) details"

Reply:
line 438: 

=>Use this as reference

Write-Host "[Step 3 of 4] Back up Container Registry(s) details"
    Write-Host $([Constants]::SingleDashLine)

    if ([String]::IsNullOrWhiteSpace($FilePath))
    {     
        # Backing up Container Registry(s) details.
        $backupFile = "$($backupFolderPath)\ContainerRegistryWithoutSecurityScanningEnabled.csv"

        $containerRegistryWithoutSecurityScanningEnabled | Export-CSV -Path $backupFile -NoTypeInformation

        Write-Host "Container Registry(s) details have been backed up to [$($backupFile)]." -ForegroundColor $([Constants]::MessageType.Update)
        Write-Host $([Constants]::SingleDashLine)
    }
    else
    {
        Write-Host "Skipped as -FilePath is provided." -ForegroundColor $([Constants]::MessageType.Warning)
        Write-Host $([Constants]::SingleDashLine)
    }